bridges/TheYeteeBridge.php

```diff
@@ -0,0 +1,47 @@
+<?php
+class TheYeteeBridge extends BridgeAbstract {
+	
+	const MAINTAINER = 'Monsieur Poutounours';
+	const NAME = 'TheYetee';
+	const URI = 'https://theyetee.com';
+	const CACHE_TIMEOUT = 14400; // 4 h 
+	const DESCRIPTION = 'Fetch daily shirts from The Yetee';
+	
+	public function collectData(){
+		
+		$html = getSimpleHTMLDOM(self::URI)
+			or returnServerError('Could not request The Yetee.');
+			
+		$div = $html->find('div');
+		foreach($div as $element) {
+			
+			if($element->class === 'hero-col')
```

Comment:
Using the selector
```PHP
 $html->find('.hero-col');
```
will allow you to find divs with such ID without having to loop through all the divs in the page